projects/ms-adal-angular6/src/ms-adal-angular6.service.ts

```diff
@@ -10,7 +10,10 @@ export class MsAdalAngular6Service {
   private context: adal.AuthenticationContext;
 
   constructor(@Inject('adalConfig') private adalConfig: any) {
-    this.context = adalLib.inject(adalConfig);
+    if (typeof adalConfig === 'function') {
```

Comment:
@unsafecode  - thanks a ton for adding this capability! Could you please update the 'Consumption of the library' section in Readme.md file so that it will help others who are looking for a similar capability as well?

Reply:
@manishrasrani Done 👍